### DISCUSSION TITLE
Skywalking 论坛的参考，以及一个疑问

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#75] [6️⃣ 提问题] [answered]
Skywalking 也是较早启用论坛功能地社区，有兴趣的可以参考下~

- https://github.com/apache/skywalking/discussions

同时，看到他们的论坛页面上有个排行榜，不知道是怎么出现的？是不是根据 QA 的贡献来的呢？

![image](https://user-images.githubusercontent.com/15010826/172030971-88010f93-e39b-4428-bc62-b19847e9283c.png)

![image](https://user-images.githubusercontent.com/15010826/172031010-fa89f961-8a4c-4edb-b4bb-4f4e22ce24f6.png)

**Comment by xgdyp** [comment, marked as answer]
I counted the list with a filter "sort:top created:>=2022-05-06 is:answered " and it has 28 records, which equals the count of the "Most helpful" module. Then I matched some of the names with the answered-by field and it matches too.
So, the value of "Most helpful" depends on how many answers were "Marked as answer".

**Reply by will-ww** [reply]
Great and congratulations ~

![image](https://user-images.githubusercontent.com/15010826/172075094-cd2ad9bd-5365-43e0-aba7-2ab1edf397f6.png)
